plugins/vpp/ifplugin/nat_config.go

```diff
@@ -247,6 +262,33 @@ func (plugin *NatConfigurator) ModifyNatGlobalConfig(oldConfig, newConfig *nat.N
 		return err
 	}
 
+	// Virtual reassembly IPv4
+	if !isDefaultVr(oldConfig.VirtualReassemblyIpv4) && isDefaultVr(newConfig.VirtualReassemblyIpv4) {
```

Comment:
This should compare only old and new value, since currently it unnecessarily calls set even if it's unchanged (non-default).